cthulhu/cthulhu/manager/eventer.py

```diff
@@ -316,6 +316,11 @@ def _on_osd_map(self, fsid, new, old):
         created_osds = new_osd_ids - old_osd_ids
 
         def osd_event(severity, msg, osd_id):
+            osd_uuid = ''
+            for osd in old.data['osds'] + new.data['osds']:
```

Comment:
yes, in latest patch am sending the osd_uuid to the function osd_event() removing the overhead that is seen above.